rest-service/manager_rest/manager_exceptions.py

```diff
@@ -429,6 +429,18 @@ def __init__(self, *args, **kwargs):
         )
 
 
+class BlueprintInUseError(ManagerException):
```

Comment:
Don't we already have something like this error when you try to delete a blueprint that an existing deployment uses?

Reply:
Nope, there isn't